lib/processors/newsprocessor.py

```diff
@@ -0,0 +1,147 @@
+from tornado import gen
+from .lib.utils import process_api_handler
+
+import re
+import random
+from .lib.baseprocessor import BaseProcessor
+
+
+class NewsProcessor(BaseProcessor):
+    name = 'news_processor'
+
+    def __init__(self):
+        super().__init__()
+
+    def fb_proxy(self, prof):
+        if 'political' in prof:
```

Comment:
What do you mean magic number codes? We just have to make sure it's in the the 0-5 range so we can give them one of the 6 articles. I don't think it REALLY matters if the article directly fits their political views.